Recon/src/Recon/OverlayProducer.cxx

```diff
@@ -395,7 +398,7 @@ void OverlayProducer::onProcessStart() {
   }
 
   // replace by this line once the corresponding tweak to EventFile is ready:
-  overlayFile_ = std::make_unique<framework::EventFile>(overlayFileName_, true);
+  overlayFile_ = std::make_unique<framework::EventFile>(params_, overlayFileName_, true);
```

Comment:
Did you try running this? I'm 90% sure that this will fail because the `EventFile` constructor is looking for the parameters `compressionSetting` and `tree_name`, but the python config class for the overlay producer doesn't define those.

Reply:
Good catch. I forgot this was done independently of ldmxcfg. I've updated this and seems to run now.